src/libcore/ops.rs

```diff
@@ -95,6 +95,16 @@ use fmt;
 #[stable(feature = "rust1", since = "1.0.0")]
 pub trait Drop {
     /// A method called when the value goes out of scope.
+    ///
+    /// When this method has been called, `self` has not yet been deallocated.
+    /// If it were, `self` would be a dangling reference.
+    ///
+    /// After this function is over, the memory of `self` will be deallocated.
+    ///
+    /// # Panics
+    ///
+    /// Given that a `panic!` will call `drop()` as it unwinds, any `panic!` in
+    /// a `drop()` implementation will likely abort.
```

Comment:
I kind of want this to say more. It's still not obvious why panicking during drop results in abort. Is there additional material we can link to that explains exception safety and double-panic?

This could be more emphatic, like

> *Implementations of `drop` must not panic*. Because drop may be called automatically called while the callstack is unwinding after a panic, panicking during drop is likely to result in a "double panic" (panic during unwinding). For reasons (link), *a double panic will cause the process to abort*. Therefore, `Drop` implementations must take care to not panic, alternatives to panicking in `drop` include ... (link).